lang/rs/wireguard/patch/0001-Integarte-CNDP-Cloud-Native-Data-Plane-with-Wireguar.patch

```diff
@@ -99,7 +99,6 @@ index 802feda..7ca64af 100644
 +**/bindings.rs
```

Comment:
I don't think we need to update the wireguard patches. The new wireguard patch uses Rust CNE library . So `pktdev_port_remove()` won't be called from wireguard

Reply:
Manoj stated we do not need to update the patch file, so reverted the changes.